es6-classes/circle.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    super(radius);
+    this.radius = radius;
+    this.area = Math.PI * this.radius * this.radius;
+    this.perimeter = 2 * Math.PI * this.radius;
```

Comment:
You should be passing this into `super` to create these properties, not creating them directly.